low-level/sspl-ll.spec

```diff
@@ -140,6 +141,13 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
 [ -f $SSPL_DIR/sspl_init ] ||
     ln -s $SSPL_DIR/bin/sspl_provisioner_init $SSPL_DIR/sspl_init
 
+# Creating softlink uinder site-packages to use sspl module easier way.
```

Comment:
Same code in https://github.com/Seagate/cortx-monitor/pull/279/files
Remove if merged after post_install PR